tools/SetupFlow/DevHome.SetupFlow/Styles/AppManagement_ThemeResources.xaml

```diff
@@ -0,0 +1,91 @@
+<!-- Copyright (c) Microsoft Corporation and Contributors. -->
+<!-- Licensed under the MIT License. -->
+
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+    xmlns:controls="using:DevHome.SetupFlow.Controls">
+    <x:Double x:Key="AppManagementIconDisabledOpacity">0.4</x:Double>
+
+    <!-- Package title style -->
+    <Style x:Key="AppManagementPackageTitleTextBlock" TargetType="TextBlock" BasedOn="{StaticResource BodyTextBlockStyle}">
+        <Setter Property="TextTrimming" Value="CharacterEllipsis" />
+        <Setter Property="Foreground" Value="{ThemeResource TextFillColorPrimary}" />
+        <Setter Property="MaxLines" Value="1" />
+    </Style>
+
+    <!-- Package description style -->
+    <Style x:Key="AppManagementPackageDescriptionTextBlock" TargetType="TextBlock" BasedOn="{StaticResource CaptionTextBlockStyle}">
+        <Setter Property="TextTrimming" Value="CharacterEllipsis" />
+        <Setter Property="Foreground" Value="{ThemeResource TextFillColorSecondary}" />
+        <Setter Property="MaxLines" Value="1" />
+    </Style>
+
+    <!-- Selectable package template -->
+    <ControlTemplate x:Key="AppManagementSelectablePackageTemplate">
+        <Grid>
+            <Grid.Resources>
+                <ResourceDictionary>
+                    <converters:BoolToObjectConverter x:Key="LearnMoreInstalledConverter">
+                        <converters:BoolToObjectConverter.FalseValue>
+                            <HyperlinkButton Name="LearnMoreButton" Command="{Binding LearnMoreCommand,Mode=OneWay}">
+                                <TextBlock x:Uid="ms-resource:///DevHome.SetupFlow/Resources/LearnMore"/>

```

Comment:
```suggestion
                                <TextBlock x:Uid="LearnMore" />
```

Reply:
We have to use an absolute path here, maybe that's because we're referencing this file in `App.xaml`? (by pointing to SetupFlow_ThemeResources.xaml)